components/Alert/alert.css

```diff
@@ -0,0 +1,33 @@
+@import url("/style.css");
+
+.container-alert {

```

Comment:
I think this is not part of your component styling. you can remove it from this file and keep it in style.css